nodepool/files/nodepool.yml

```diff
@@ -66,6 +71,11 @@ providers:
             flavor-name: performance2-15
             key-name: jenkins
             console-log: true
+          - name: rpco-14.2-xenial-base
+            diskimage: rpco-14.2-xenial-artifacts
+            flavor-name: 7
```

Comment:
This flavor comes from https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/rpc_openstack.yml#L357 which is where the new image would be used.